src/Middleware/CORS/src/Infrastructure/CorsPolicyBuilder.cs

```diff
@@ -224,6 +224,11 @@ public CorsPolicyBuilder SetIsOriginAllowedToAllowWildcardSubdomains()
         /// <returns>The constructed <see cref="CorsPolicy"/>.</returns>
         public CorsPolicy Build()
         {
+            if (_policy.AllowAnyOrigin && _policy.SupportsCredentials)
```

Comment:
What about `Combine`?

Reply:
You have to call `Build` to get to the the result of `Combine`.